Sources/Customization/Protocols/OPTDatafileHandler.swift

```diff
@@ -25,18 +25,35 @@ public protocol OPTDatafileHandler {
     init()
     
     var endPointStringFormat: String { get set }
+    
+    /**
+     Save an interval for periodic polling
+     
+     - Parameter sdkKey: sdk key of the datafile to download
+     - Parameter interval: interval in secs (> 0)
+     */
+    func setPeriodicInterval(sdkKey: String, interval: Int)
+    
+    /**
+     Check if periodic polling has been set
+     
+     - Parameter sdkKey: sdk key of the datafile to download
+     - Returns: true if set
+     */
+    func hasPeriodicInterval(sdkKey: String) -> Bool
```

Comment:
why is this needed? since this is only used internally by the sdk and i don't see it being used publicly for any use case.

Reply:
This is used by OptimizelyClient to determine if config will be updated on datafile download. Since we now open DatafileHandler, this interface is added.